docs/BMI-Installation.md

```diff
@@ -0,0 +1,287 @@
+
+```
+service tftp
+{
+        socket_type             = dgram
+        protocol                = udp
+        wait                    = yes
+        user                    = root
+        server                  = /usr/sbin/in.tftpd
+        server_args             = /tftpboot
+        disable                 = no
+        per_source              = 11
+        cps                     = 100 2
+        flags                   = IPv4
+}
+```
+
+* Create a folder `/tftpboot` this should match whatever you gave in `server_args.` mostly it will be `tftpboot`:
+  
+```
+$ sudo mkdir /tftpboot
+$ sudo chmod -R 777 /tftpboot
+$ sudo chown -R nobody /tftpboot
+```
+  
+* Restart the `xinetd` service:  `$ sudo service xinetd restart`
+
+
+
+The TFTP server should be up and running. For more information click [here](http://askubuntu.com/questions/201505/how-do-i-install-and-run-a-tftp-server)
+
+#### Hardware Isolation Layer (HIL)
+
+To setup HIL, you can read the [HIL documentation](http://hil.readthedocs.io/en/latest/)
+
+### Installling
+* Clone this repository to your home folder and change to `dev` branch
+```
+$ git clone https://github.com/CCI-MOC/ims
+$ git checkout dev
+```
+* Run setup.py and install python-cephlibs
+```
+$ sudo python setup.py install
+$ pip install python-cephlibs
+```
+
+That's it. Installation is done!
+***
+## Running
+
+
+### Configuration
+
+The template for the config is [here](https://github.com/CCI-MOC/ims/blob/dev/bmi_config.cfg)
+
+This section describes each config section along with sample  
+
+**bmi section**  
+* uid is used to prevent clashing of BMIs that use the same Ceph Pool. It is the responsibility of the admin to ensure that uid is unique.
+* service should be set to True if BMI is deployed as a service. Set it to False for now as there are some glitches.
+```
+# this section is for basic bmi settings
+[bmi]
+# uid is given so that images dont clash in ceph pool
+uid = 5
+service = False
+```  
+
+**db section**  
+* url is the path of the sqlite db
+```
+# this section is for db settings
+[db]
+url = /home/user/bmi.db
+```  
+
+**filesystem section**  
+Tells which Filesystem to load (right now, we only support ceph)
+
+* id is the ceph id which has access to the pool
+* pool is the ceph pool to use for storing images
+* conf_file is the path to the Ceph conf file
+* keyring is the path to the keyring file
+```
+# This section is filesystem related config
+[fs]
+id = bmi
+pool = bmi
+conf_file = /etc/ceph/ceph.conf
+keyring = /etc/ceph/bmi.key
+```
+
+**Driver section**
+* net_isolator is the name of the network isolator to load
+* iscsi is the iscsi driver to load
+* fs is the filesystem to use
+```
+# This section is for hil related config
+[driver]
+fs = ims.einstein.fs.ceph.driver
+net_isolator = not used
+iscsi = ims.einstein.tgt.driver
+```  
+
+**iscsi section**
+* ip is the ip of iscsi server on the provisioning network
+* password is the sudo password for the VM (Will be removed)
+```
```

Comment:
It already says "will be removed". @chemistry-sourabh if it's already deprecated then I'll just remove this line.